app/config.go

```diff
@@ -62,6 +62,7 @@ type Config struct {
 	DailyActivesRetention       int
 	WeeklyActivesRetention      int
 	ErrorReporter               ops.ErrorReporter
+	Logger                      logrus.FieldLogger
```

Comment:
This is the main interface provided by logrus - so its possible to provide your own implementation as a dependee